web/src/main/java/org/akaza/openclinica/control/submit/ListStudySubjectTableFactory.java

```diff
@@ -1334,4 +1365,47 @@ private String formatDate(Date date) {
         return sdf.format(date);
     }
 
+    public String getPortalURL() {
+        Properties p = new Properties();
+        InputStream inpStream = null;
+        try {
+                inpStream = this.getClass().getClassLoader().getResourceAsStream("datainfo.properties");
+                p.load(inpStream);
+                inpStream.close();
+        } catch (Exception e) {}
+        return p.getProperty("portalURL");
+         
+    }
+
+    public void setOcuiParameter() {
+        String host = null;
+        String currentAddress = servletRequest.getRequestURL().toString().split("OpenClinica")[0];
+        String tmpAddress = null;
+        try {
+            Class.forName("org.postgresql.Driver");
+
+            /* need to be modified based on database location */
+            connection = DriverManager.getConnection("jdbc:postgresql://127.0.0.1:5432/epro", "epro","epro");
```

Comment:
@kkrumlian is this correct? or should I add a new entry in datainfo.properties for this? (say let's call it portalDatabaseURL)